pkg/apis/v1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Did we mark this version as the storage version?

Reply:
Yes, the NodePool and NodeClaim for the v1 APIs are marked as the storage versions